app/assets/stylesheets/homepage.scss

```diff
@@ -0,0 +1,47 @@
+$coral: #ec4339;
+$white: #ffffff;
+
+.title-box {
+  background-color: $coral;
+  font-family: 'Alegreya Sans SC', sans-serif;
+  color: $white;
+  padding: 10px;
+  font-weight: bold;
+  letter-spacing: 1.25px;
+  font-size: 60px;
+  font-size: 5vw;
+ }
+
+.vid-image {
+  width: -webkit-fit-content;
+  height: -webkit-fit-content;
+  width: -moz-fit-content;
+  height: -moz-fit-content;
+  position: relative;
+  background-position-x: 50%;
+}
+
+.vid-overlay {
+  bottom: 0;
+  color: $white;
+  display: none;
+  float: left;
+  left: 0;
+  padding-top: 23%;
+  position: absolute;
+  right: 0;
+  text-align: center;
+  top: 0;
+  font-size: 1.5em;
+  font-weight: bolder;
+  text-shadow: 4px 4px 10px black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.